src/components/pages/dashboard/panels/map/mapPanel.js

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Microsoft. All rights reserved.
 
 import React, { Component } from 'react';
+import update from 'immutability-helper';
```

Comment:
Other imports start with upper case letter

Reply:
This is importing a function. Other imports are importing classes and react components.